### PR TITLE
[14.0][FIX] t3954  The value of a variable cannot be assigned to several records same time

### PR DESCRIPTION
**Pull request by dessanhemrayev**
Add migration script for change field name in model(table) 
Change module version

### CODE REVIEW

File: cetmix_tower_server/tests/test_variable.py [hunk 1]
```diff
@@ -659,3 +662,63 @@ def test_get_by_variable_reference(self):
...
+                "ssh_username": "admin1",
+                "ssh_password": "password1",
+                "ssh_auth_mode": "p",
+            }
+        )
+
+        self.env["cx.tower.variable.value"].create(
+            {
+                "variable_id": variable.id,
+                "value_char": "Branch = Main",
```
Comment (ivs-cetmix) [inline cetmix_tower_server/tests/test_variable.py]: This is very strange value although correct.
Final expression will look like `Test Variable Custom = Branch = Main`
So you should find some more relevant value here. Even if this is just a test)

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
Some small changes otherwise LGTM

**Review by GabbasovDinar — APPROVED** [review]

File: cetmix_tower_server/tests/test_variable.py (hunk 1 above)
Comment (dessanhemrayev) [reply cetmix_tower_server/tests/test_variable.py]: Thanks for review, I'll fix it

**Review by ivs-cetmix — APPROVED** [review]
Code review LGTM

**Review by Aldeigja — APPROVED** [review]
Functional LGTM

**Comment by CetmixGitDrone** [comment]
Hey, thanks for contributing! Proceeding to merge this for you.
Prepared branch [14.0-dev-ocabot-merge-pr-107-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-107-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at cca1f19d19295d5abe45582ecfc93c6d35ab57cd. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**